src/configure

```diff
@@ -1317,18 +1317,29 @@ elif [ "`uname`" == "Linux" ]; then
     if [ -z "$OPENBLASROOT" ]; then
       failure "Must specify the location of OPENBLAS with --openblas-root option (and it must exist)"
     fi
-    if [ ! -f $OPENBLASROOT/lib/libopenblas.so ]; then
+    if [ -f $OPENBLASROOT/lib/libopenblas.so ]; then
+      OPENBLASLIBDIR=$OPENBLASROOT/lib
+    elif [ -f $OPENBLASROOT/lib64/libopenblas.so ]; then
+      OPENBLASLIBDIR=$OPENBLASROOT/lib64
+    else
       failure "Expected to find the file $OPENBLASROOT/lib/libopenblas.so"
     fi
+    if [ -f $OPENBLASROOT/include/openblas_config.h ] ; then
+      OPENBLASINCDIR=$OPENBLASROOT/include
+    elif [ -f $OPENBLASROOT/include/openblas/openblas_config.h ] ; then
```

Comment:
Could you comment that this is the debian-based install path, and that the former is (I assume) the rpm install path.

Reply:
Actually, never mind, on a Centos7 machine and a Ubuntu 16.04 machine, I get /usr/include/openblas after installing the appropriate packages. What is this case `if [ -f $OPENBLASROOT/include/openblas_config.h ] ; then` handling? Is it when openblas is installed via kaldi's tools/ directory?

Reply:
The build fails.  I wonder if that is why it's failing to build at kaldi-asr. Hmm

Reply:
I don't think search for openblas_config.h is appropriate in the case where it's a system package.  I believe the dev package may provide cblas.h and lapacke.h.   Just replacing openblas_config.h with cblas.h in all these checks should be fine, and may allow it to build on travis.